src/components/Clock.jsx

```diff
@@ -192,17 +195,27 @@ export class Clock extends React.Component {
 			d = 360 + d
 		}
 
+		//sometimes touch bleeds into please it shoudln't
+		const r = Math.sqrt(x * x + y * y)
+		if (r > CLOCK_RADIUS && this.dragCount < 2){
+			return false;
+		}
+
 		const unit = this.props.unit
-		const selected = Math.round( d / 360 * CLOCK_DATA[unit].increments )
+		const selected = Math.round( d / 360 * (CLOCK_DATA[unit].increments / (isCoarse ? CLOCK_DATA[unit].coarseMultipiler : 1)) )
 
 		if (unit === 'hour'){
-			this.props.changeHour(selected, canChangeUnit)
+			this.props.changeHour(selected * (isCoarse ? CLOCK_DATA[unit].coarseMultipiler : 1), canChangeUnit)
```

Comment:
Just so i understand correctly, drag count is used so that you need to drag 2+ units to select non-coarse (ie: 1,2,3,4) values?

Reply:
Yes, one of the issues and the project this was emulating snaps the first tap to 5 minute increments. The most resonable way I could see to handle this was to check if the user actually did some dragging before allowing off fine selection of units.